Example/Backpack/View Controllers/SettingsViewController.swift

```diff
@@ -21,18 +21,79 @@ import Backpack
 
 class SettingsViewController: UITableViewController {
     @IBOutlet weak var closeButton: UIBarButtonItem!
+    @IBOutlet weak var enableThemeSwitch: Switch!
+    var showThemeList: Bool = false
+    @IBOutlet var selectableCells: [UITableViewSelectableCell]!
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         closeButton.tintColor = Color.blue500
         closeButton.target = self
         closeButton.action = #selector(SettingsViewController.btnAction)
+        enableThemeSwitch.addTarget(self, action: #selector(SettingsViewController.themingToggled), for: .touchUpInside)
+
+        let theme = Settings.sharedSettings.activeTheme
+        updateUi(theme: theme)
     }
 
     @objc
     func btnAction() {
         self.dismiss(animated: true, completion: nil)
     }
 
+    @objc
+    func themingToggled() {
```

Comment:
Can we call this `didToggleTheming`?

Reply:
We should use the sender here to set the value rather than getting it from `enableThemeSwitch`